CHANGELOG.md

```diff
@@ -5,16 +5,40 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+
+
 ## [Unreleased]
+### Added
+
+### Changed
+
+### Fixed
+
+### Removed
```

Comment:
I would remove those empty sections. If you want to keep `## [Unreleased]` is okay, but the sub sections should be added only if there is at least one item (if nothing is removed, the `removed` section should be present, etc.): that is valid not only for `Unreleased`.

So basically I would delete from line 11 to line 17.

Reply:
LGTM. Done.